e2e/cypress/integration/multi_team_and_dm/close_current_dm_redirects_spec.js

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+describe('Direct messages: redirections', () => {
+    let testUser;
+    let secondDMUser;
+    let firstDMUser;
+    let testTeam;
+    let offTopicChannelUrl;
+
+    before(() => {
+        // # Create a new team
+        cy.apiInitSetup().then(({team, user}) => {
+            testUser = user;
+            testTeam = team;
+
+            offTopicChannelUrl = `/${testTeam.name}/channels/off-topic`;
+
+            cy.apiCreateUser().then(({user: createdUser}) => {
+                firstDMUser = createdUser;
+                cy.apiAddUserToTeam(team.id, firstDMUser.id);
+            });
+
+            // # Create a second test user
+            cy.apiCreateUser().then(({user: createdUser}) => {
+                secondDMUser = createdUser;
+                cy.apiAddUserToTeam(testTeam.id, secondDMUser.id);
+            });
+        });
+    });
+
+    it('MM-T453 Closing a direct message should redirect to last viewed channel', () => {
```

Comment:
Looks like this is combination of step 1 and step 2 which I think makes sense. I'll revise the test case. Not documented yet but in case of multiple steps, we need to change test keys into `[test_case_id]_[step_number]`.
```suggestion
    it('MM-T453_1 Closing a direct message should redirect to last viewed channel', () => {
```